api/lib/infrastructure/repositories/competence-repository.js

```diff
@@ -7,44 +7,52 @@ const competenceDatasource = require('../datasources/airtable/competence-datasou
 const knowledgeElementRepository = require('./knowledge-element-repository');
 const scoringService = require('../../domain/services/scoring/scoring-service');
 const { NotFoundError } = require('../../domain/errors');
+const { FRENCH_FRANCE, ENGLISH_SPOKEN } = require('../../domain/constants').LOCALE;
 
 const PixOriginName = 'Pix';
 
-function _toDomain(competenceData, areaDatas) {
+function _toDomain(competenceData, areaDatas, locale) {
   const areaData = competenceData.areaId && _.find(areaDatas, { id: competenceData.areaId });
   return new Competence({
     id: competenceData.id,
-    name: competenceData.name,
+    name: _getTranslatedText(locale, { frenchText: competenceData.nameFrFr, englishText: competenceData.nameEnUs }),
```

Comment:
J'aurais tendance à extraire cette ligne pour gagner en lisibilité 
```suggestion
const translastedName = _getTranslatedText(locale, { frenchText: competenceData.nameFrFr, englishText: competenceData.nameEnUs });

return new Competence({
    id: competenceData.id,
    name: translatedName,
```